config/default/manager_gardener_secret_patch.yaml

```diff
@@ -19,6 +19,8 @@ spec:
             - /manager
           args:
             - --gardener-kubeconfig-path=/gardener/credentials/kubeconfig
+            - --gardener-project-name=kyma-dev
```

Comment:
Should we already capture somewhere (maybe in some issue or documentation) that this should be overridden for other environments?